File: August/August_26_fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int):
        l=[]
        for i in range(1,n+1):
            if i%3==0 and i%5==0:
                l.append("FizzBuzz")
                continue
            elif i%3==0:
                l.append("Fizz")
            elif i%5==0:
                l.append("Buzz")
            else:
                l.append(str(i))
        return l
        

File: August'20/August_26_Fizz_Buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> s;
        for(int i=1; i<n+1; i++){
            if(i%3==0 && i%5==0){
                s.push_back("FizzBuzz");
            }else if(i%3==0){
                s.push_back("Fizz");
            }else if(i%5==0){
                s.push_back("Buzz");
            }else{
                string st = to_string(i);
                s.push_back(st);
            }
        }
        
        return s;
    }
};
